week1/plot_trips.R

```diff
@@ -47,16 +75,33 @@ trips_with_weather <- inner_join(trips, weather, by="ymd")
 
 # plot the number of trips as a function of the minimum temperature, where each point represents a day
 # you'll need to summarize the trips and join to the weather data to do this
-
+by_day <- group_by(trips_with_weather, date) %>% summarize(count = n())
+by_t_and_d <- group_by(trips_with_weather, ymd, tmin) %>% summarize()
+new_col <- select(by_day, count)
+x <- cbind(by_t_and_d, new_col)
+ggplot(data = x, mapping = aes(x = tmin, y = ymd)) + geom_point(aes(size = count))
 # repeat this, splitting results by whether there was substantial precipitation or not
 # you'll need to decide what constitutes "substantial precipitation" and create a new T/F column to indicate this
-
+trips_with_weather_and_p <- trips_with_weather
+trips_with_weather_and_p$sub_prcp <- with(trips_with_weather, ifelse(prcp > 1.50, "T", "F"))
```

Comment:
This would be wayyyy easier with mutate, and also you can store the data as a boolean (as F and T without quotes) by not including the `ifelse` part and just inserting the `prcp > 1.50` directly as a column like

`trips_with_weather_and_p$sub_prcp <- trips_with_weather$prcp > 1.50`

But again, it's even cleaner if you do it with mutate in a pipe.